.travis.yml

```diff
@@ -0,0 +1,45 @@
+language: python
+python:
+  - 3.4
+  - 3.5
```

Comment:
[no](https://travis-ci.org/jayvdb/coala/builds/182831634).  well, `3.6` hasnt been added to Travis yet. maybe they have it available under a different label, but I havent looked.  we can add it soon, I am sure.